client/connect-account-page/index.js

```diff
@@ -63,29 +136,63 @@ const ConnectPageOnboarding = () => {
 					onClick={ handleSetup }
 					href={ wcpaySettings.connectUrl }
 				>
-					{ __( 'Set up', 'woocommerce-payments' ) }
+					{ strings.button }
 				</Button>
 			</p>
 		</>
 	);
 };
 
+const ConnectPageOnboardingSteps = () => {
+	return (
+		<>
+			<h2>{ strings.stepsHeading }</h2>
+			<div className="connect-page-onboarding-steps">
+				<div className="connect-page-onboarding-steps-item">
+					<StepNumber>1</StepNumber>
+					<h3>{ strings.step1[ 0 ] }</h3>
+					<p>{ strings.step1[ 1 ] }</p>
```

Comment:
The `0` and `1` indices feel like magic numbers. Without going to the separate file where they are defined, I have no guess if `0` and `1` are the correct indices to use.

Reply:
Thanks for highlighting this. thought it was following a convention in WC Pay here as well, but on closer look it seems like it is only used when there is a sequence of paragraphs.

I've fixed it in 4e21671